src/stdio_mgr/stdio_mgr.py

```diff
@@ -346,37 +323,10 @@ def __new__(cls, in_str="", close=True):
 
         return self
 
-    @property
-    def stdin(self):
-        """Return capturing stdin stream."""
-        return self[0]
-
-    @property
-    def stdout(self):
-        """Return capturing stdout stream."""
-        return self[1]
-
-    @property
-    def stderr(self):
-        """Return capturing stderr stream."""
-        return self[2]
-
-    def __enter__(self):
-        """Enter context, replacing sys stdio objects with capturing streams."""
-        self._prior_streams = (sys.stdin, sys.stdout, sys.stderr)
-
-        super().__enter__()
-
-        (sys.stdin, sys.stdout, sys.stderr) = self
-
-        return self
-
-    def __exit__(self, exc_type, exc_value, traceback):
-        """Exit context, closing files and restoring state of sys module."""
-        (sys.stdin, sys.stdout, sys.stderr) = self._prior_streams
-
+    def close(self):
+        """Close files only if requested."""
         if self._close:
-            super().__exit__(exc_type, exc_value, traceback)
+            return super().close()
```

Comment:
Is this in case mocks/wraps have a return value?

Reply:
The `return` is unnecessary.  Automatic fingers.

Reply:
This change dropped from this PR